include/ada/url_aggregator-inl.h

```diff
@@ -404,7 +405,7 @@ inline void url_aggregator::append_base_username(const std::string_view input) {
     components.hash_start += difference;
   }
 #if ADA_DEVELOPMENT_CHECKS
-  std::string username_after = std::string(get_username());
+  std::string const username_after{get_username()};
```

Comment:
The original is ugly and worth fixing...
```
std::string username_after = std::string(get_username());
```

But why do...

```
std::string const username_after{get_username()};
```

when you can do...

```C++
std::string username_after{get_username()};
```

which is nicer.

Reply:
I'm sorry, but `const` has more value for me than simplicity. When I see a variable with no const, I expect it to change somewhere in the code.